example_configs/quay.md

```diff
@@ -0,0 +1,26 @@
+# Configuration for Quay
+
+Quay is a container image registry that enables you to build, organize, distribute, and deploy containers.
+
+The official LDAP configuration documentation is located [here](https://docs.projectquay.io/config_quay.html#config-fields-ldap).
+
+For standalone deployments of Project Quay, the core configuration is primarily set through the `config.yaml` file.
+
+If you install Project Quay on OpenShift Container Platform / OKD using the Project Quay Operator, the configuration resides in the `config bundle secret`. 
+
+This example assists with a basic LDAP configuration.
+
+```yaml
+AUTHENTICATION_TYPE: LDAP
+LDAP_ADMIN_DN: cn=admin,ou=people,dc=example,dc=com
```

Comment:
What role do you need for the bind user? Admin, read-only, allow password changes?

Reply:
The bind user must be able to login and view the records for all user accounts

Reply:
Sounds good! Can you use a different bind user instead of admin, and document that it should be in `lldap_atrict_readonly`?

Reply:
I modified the example configuration, used a different bind user and mentioned the user should be part of `lldap_strict_readonly` group.